test/models/loan_test.rb

```diff
@@ -10,13 +10,42 @@ class LoanTest < ActiveSupport::TestCase
 
   test "it is not valid without a project id" do
     loan = build(:loan, project_id: nil)
+    loan.update_attributes(project_id: nil)
 
     assert loan.invalid?
   end
 
   test "it is not valid without an order id" do
-    loan = build(:loan, order_id: nil)
+    loan = create(:loan)
+    loan.update_attributes(order_id: nil)
+
+    assert loan.invalid?
+  end
+
+  test "it is not valid without an amount" do
+    loan = create(:loan)
+    loan.update_attributes(amount: nil)
+
+    assert loan.invalid?
+  end
```

Comment:
I thought previously there was a validation that set loan amount to a certain default?